static/src/stylesheets/module/content/tones/_tone-media.scss

```diff
@@ -48,6 +48,13 @@
         border-color: $neutral-2;
     }
 
+    .caption--img {
+        a {
+            color: $neutral-7;
+            border-bottom: 1px solid $neutral-2;
+        }
+    }
+
```

Comment:
Are there other links on the page whose styles we could share/re-use here? Links in the standfirst are the same, I think?